src/core/caniuse.js

```diff
@@ -0,0 +1,218 @@
+/*
+Module: "core/caniuse"
+Adds a caniuse support table for a "feature" #1238
+
+`conf.caniuse = { feature: key }` =>
+  .. the table is added before Copyright
+`<section data-caniuse=key>` =>
```

Comment:
I don't think we should do this yet. There are a number of issues with the current approach that will make the HTML invalid. We should work out what this will look like first, then add this in.

Reply:
sure